### ISSUE TITLE
MS Office Failing due to unchecked APIs

### ISSUE THREAD

**Issue by mattpen** [phetsims/phet-office-mix#4]
According to Microsoft, our app is failing submission due to unchecked use of webkitAudioContext().

I searched for uses of new webkitAudioContext() and found several in john-travoltage, one in phet-io, and one in sherpa (used only by molecule-shapes).

We are using webkitAudioContext() in `vibe/js/Sound.js`, but we appear to checking correctly for its presence here.

**Comment by mattpen** [comment]
One problem in remedying this problem is that we cannot view console output errors while it is running in PPT for Mac.

**Comment by samreid** [comment]
Can you elaborate on "our app"?  Are we submitting one composite app, with all of the sims, or is one sim in particular failing?

**Comment by mattpen** [comment]
@samreid 

We are submitting a manifest file, which points to an office store app hosted at https://phet.colorado.edu/office-mix/store.html.  You can [run the app in a browser here](https://labsjs.blob.core.windows.net/sdk/LabsJS-1.0.4/labshost.html?lab=https://phet.colorado.edu/office-mix/store.html?PostMessageLabHost).

To view the app in Powerpoint on a Mac, you'll need a copy of [the manifest](https://github.com/phetsims/phet-office-mix/blob/master/phet-manifest.xml) and then follow [these instructions](https://dev.office.com/docs/add-ins/testing/sideload-an-office-add-in-on-ipad-and-mac#sideload-an-add-in-on-office-for-mac).

Basically, the app itself is working, but when we load some of the simulations into memory they are getting stuck at the splash screen.  MS suggests this is due to unchecked calls to the WebKitAudioContext constructor, but this is happening on sims that I don't think actually call that function, Arithmetic for instance.  Most of our sims use the Sound.js api, which checks for the presence of the constructor before calling it ([code](https://github.com/phetsims/vibe/blob/3230caa4e15496ea9d7d1700e368b2ef5c4228b7/js/Sound.js#L29)).

**Comment by mattpen** [comment]
I'll go ahead and add checks to the webkitAudioContext()

**Comment by samreid** [comment]
Unassigning while @mattpen pursues the strategy in the preceding comment.

**Comment by mattpen** [comment]
The app was published after MS pushed updates to Powerpoint for OSX.  This issue does not appear to be affecting our sims elsewhere, and I couldn't locate any instances of unchecked calls to this API anyway, so I'm going to close for now.
